### ISSUE TITLE
Only the last referenced document of an array is clickable

### ISSUE THREAD

**Issue by WeeJeWel** [SoftwareBrothers/adminjs#695]
**Describe the bug**
I have a mongoose model Thing with a property users. This is an array of objects with each a ref to User using an ObjectId.

In the front-end this translates nicely to a link with the User's email address, and I can click it to view that user.

However only the last User of the array is resolved, the others show a non-clickable ObjectId.

**Installed libraries and their versions**

* admin-bro@2.0.1
* admin-bro-expressjs@2.0.0
* admin-bro-mongoose@1.4.0

**Expected behavior**
All refs should be resolved.

**Comment by dziraf** [comment]
Can you check if the problem still exists in the latest version of `admin-bro` & `@admin-bro/mongoose` & `@admin-bro/expressjs`? `2.0.1` is a bit outdated

**Comment by WeeJeWel** [comment]
Sorry, I mentioned the incorrect versions.

```
    "@admin-bro/express": "^3.0.1",
    "@admin-bro/mongoose": "^1.1.0",
    "admin-bro": "^3.3.1",
```

**Comment by dziraf** [comment]
Could you share your mongoose schemas? I'll try to reproduce/fix it sometime this week

**Comment by WeeJeWel** [comment]
Sure!

```javascript
module.exports = mongoose.model('User', {
  email: String,
  firstname: String,
  lastname: String,
  roles: [String],
})
```

```javascript
module.exports = mongoose.model('Thing', {
  name: String,
  users: [{
    role: String,
    userId: {
      type: mongoose.ObjectId,
      ref: 'User',
    },
  }],
})
```

**Comment by dziraf** [comment]
@WeeJeWel The fix is included in `admin-bro@3.3.2-beta.2` so I'm closing this issue. Sorry for the delay in fixing this.

**Comment by WeeJeWel** [comment]
Thank you!

**Comment by WeeJeWel** [comment]
I'm sorry, but it doesn't seem to be solved for me using `admin-bro@3.3.2-beta.2`.

**Comment by dziraf** [comment]
I'll check this again next week but I reproduced the problem using your schema and this seemed to have fixed it. Have you added any custom property options in resource options?
